decidim-admin/app/controllers/decidim/admin/concerns/has_attachments.rb

```diff
@@ -23,13 +23,13 @@ def index
 
           def new
             authorize! :create, authorization_object
-            @form = form(AttachmentForm).instance
+            @form = form(AttachmentForm).from_params({}, attached_to: attached_to)
```

Comment:
Why did you need to add this?

Reply:
I need this to know the `attached_to` in the view, so I can get the `attachment_collections` associated with it and render the select.